packages/system/setup.py

```diff
@@ -0,0 +1,76 @@
+"""pi-top Python SDK."""
+
+import os
+import sys
+
+from setuptools import setup
+
+if not sys.version_info >= (3, 7):
+    raise ValueError("This package requires Python 3.7 or above")
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+__version__ = os.environ.get("PYTHON_PACKAGE_VERSION", "0.0.1.dev1")
+assert __version__ != ""
+
+__project__ = "pitop.system"
+__author__ = "pi-top"
+__author_email__ = "deb-maintainers@pi-top.com"
+
+__url__ = "https://github.com/pi-top/pi-top-Python-SDK"
+__platforms__ = "ALL"
+
+# https://pypi.org/classifiers/
+__classifiers__ = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Education",
+    "Intended Audience :: Developers",
+    "Topic :: Education",
+    "Topic :: System :: Hardware",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+
+__keywords__ = [
+    "pi-top",
+    "raspberrypi",
+    "battery",
+]
+
+__requires__ = [
+    f"pitop.common=={__version__}",
+    f"pitop.core=={__version__}",
```

Comment:
Explicitly depending on `pitop.battery` and `pitop.miniscreen` causes a circular dependency, so we must make these soft dependencies. For now I think we should leave the error message as is to avoid changing the behaviour of any packages in this PR.